Theme.md

```diff
@@ -30,3 +30,61 @@ Now, go into the settings of your store and paste the link to customize the boot
 And enjoy a new theme!
 
 ![CustomTheme](img/CustomTheme.png)
+
+## Checkout page theme
+
+It is possible to entirely customize the checkout experience of your customer with your brand.
+
+Go to you Store Settings, Checkout Experience, then follow those examples:
```

Comment:
Go to your Store Settings, Checkout Experience, Link to a custom CSS stylesheet. Create your custom theme using these examples below:

Maybe we can add something like:

Server admins can upload their .css file to [File Storage](https://github.com/btcpayserver/btcpayserver-doc/blob/master/FAQ/FAQ-ServerSettings.md#how-to-upload-files-to-btcpay) to create a CSS stylesheet link.